src/helpers/lnurl.ts

```diff
@@ -2,5 +2,5 @@ import { bech32 } from 'bech32';
 
 export const encodeLnurl = (str: string) => {
   const words = bech32.toWords(Buffer.from(str, 'utf-8'));
-  return bech32.encode('lnurl', words);
+  return bech32.encode('lnurl', words, 2000);
```

Comment:
Let's also have the character number as an env variable that is `undefined` as default.

Also take into account from the [bech32 lib](https://github.com/bitcoinjs/bech32): *It is highly recommended NOT exceed 1023 characters, as the module could only guarantee detecting 1 error.*

Reply:
I put 2k because it is the maximum size of URL. same 1k is fine anyway
And then I do not consider that I have to put as an environment variable something that is rather internal to the program